LogMonitor/docs/README.md

```diff
@@ -0,0 +1,188 @@
+# Log Monitor Documentation
+
+**Contents:**
+
+- [ETW Monitoring](#etw-monitoring)
+- [Event Log Monitoring](#event-log-monitoring)
+- [Log File Monitoring](#log-file-monitoring)
+- [Process Monitoring](#process-monitoring)
+
+## ETW Monitoring
+
+### Description
+
+ETW (Event Tracing for Windows) is a system for routing events. ETW is primarily intended for diagnostic purposes and is optimized to minimize impact on the overall system performance. Please note that ETW is different from the Windows Event Log.
+
+To monitor ETW logs using Log Monitor you require a valid _provider GUID_ (Globally Unique Identifier).
+
+The _provider GUID_ that you pass in the JSON config file provider must be registered with ETW. Log Monitor currently allows you to monitor events generated by MOF (classic) providers, manifest-based providers and Trace Logging providers. _**Note**: we do not monitor log events from WPP providers._
```

Comment:
Remove provider so that is reads: The _provider GUID_ that you pass in the JSON config file must be registered with ETW.